### PR TITLE
Add a note after windows and before dependencies (Issue #1320)

### PR DESCRIPTION
**Pull request by aas919**
Rawgit Link [Here](https://rawgit.com/aas919/aas919.github.io/add-tg-installation/#!pages/techgenius/tg-installation.md)

### CODE REVIEW

**Comment by ajGingrich** [comment]
This change seems necessary in case vagrant isn't running. I think it would be cleaner if we used just one note that referred the vagrant instructions.

File: pages/techgenius/tg-installation.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ powershell -Command "(New-Object Net.WebClient).DownloadFile('https://raw.github
 ```
 To run your community BeLL at the end of the installation, please, find the MyBeLL icon on your desktop and double click on it. It will open a Firefox browser and take you directly to your community BeLL web page.
 
+Note: If the firefox displays ```Unable to connect``` error when the MyBell icon is clicked, try the following steps in the command prompt
```
Comment (thesiege208) [inline pages/techgenius/tg-installation.md]: Minor issue, but it should be "Firefox" rather than "the firefox."

Comment (thesiege208) [inline pages/techgenius/tg-installation.md]: Minor edit, but "the firefox" should simply say "Firefox."

**Comment by aas919** [comment]
@ajGingrich I have trimmed the note to refer to Vagrant instructions in case of Unable to connect error. Please review the same.
@thesiege208 Fixed it. Thank you

**Comment by xyb994** [comment]
@aas919 It seems you haven't remove the extra "the" yet

**Comment by aas919** [comment]
@xyb994  Fixed it. Thank you

**Comment by aas919** [comment]
@thesiege208 @xyb994 Please review and merge the PR if possible. Thank you

**Pull request merged**